### PR TITLE
feat: add contests and tasks for flow 

### PR DESCRIPTION
**Pull request by KATO-Hiro**
- Add pastbook2022 tasks (A-H) to prisma/tasks.ts
- Add WUPC (wupc2019) to university contest regex and test cases
- Add missing AtCoder Others contest name labels and type test cases (geocon2013, s8pc-3, maximum-cup-2013/2018, code-festival-2014-china-open, soundhound2018, pakencamp-2018-day3, tenka1-2012-qualB, nikkei2019-final, awtf2024, xmascon19)

## リリースノート

* **新機能**
  * 典型アルゴリズム問題集（pastbook2022）の8つの問題（A～H）を追加
  * 早稲田大学プログラミングコンテスト（WUPC）に対応
  * 複数のAtCoderコンテストの表示に対応し、より多くのコンテストをサポート

* **テスト**
  * コンテスト名とタイプの検証テストを拡充

### CODE REVIEW

File: prisma/tasks.ts [hunk 1]
```diff
@@ -7667,6 +7667,62 @@ export const tasks = [
...
+    name: '一点更新・区間最小値',
+    title: 'G. 一点更新・区間最小値',
+  },
+  {
+    id: 'pastbook2022_h',
+    contest_id: 'pastbook2022',
+    problem_index: 'H',
+    name: '区間加算・区間最小値',
+    title: 'H. 区間加算・区間最小値',
+  },
```
[the remark this answers is not in the document]
Comment (KATO-Hiro) [reply prisma/tasks.ts:7725]: 現時点では難易度が未評価なのでそのままとします
また、pending という項目で処理しているため問題はないと考えています

**Review by KATO-Hiro — COMMENTED** [review]
LGTM
